openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedDrawer/activity-feed-drawer.less

```diff
@@ -19,3 +19,9 @@
     background-color: @white;
   }
 }
+
+.reserve-right-sidebar {
+  .activity-feed-editor {
+    margin-right: 64px;
+  }
+}
```

Comment:
can we use existing style or add class in existing style instead of creating new css
you will get existing style in app.less